lib/DDG/Spice/ExpandURL.pm

```diff
@@ -4,9 +4,21 @@ package DDG::Spice::ExpandURL;
 use strict;
 use DDG::Spice;
 
-triggers start => "http", "https", "expand", "expandurl", "unshorten";
-
-spice to => 'http://api.longurl.org/v2/expand?url=$1&format=json&callback={{callback}}';
+primary_example_queries "http://bit.ly/a", "untiny bit.ly/lolcatz";
```

Comment:
@MrChrisW we can remove all this metadata -- all this lives in the IA Page now. We'll be deprecating the Perl metadata entirely soon at which point these will no longer be recognized.